server-adapters/resteasy-undertow-cdi/src/main/java/dev/resteasy/embedded/server/UndertowCdiEmbeddedServer.java

```diff
@@ -69,6 +70,11 @@ public void start(final Configuration configuration) {
                 builder.setSocketOption(Options.SSL_CLIENT_AUTH_MODE, SslClientAuthMode.REQUIRED);
                 break;
         }
+        ServiceLoader<UndertowBuilderConfigurer> undertowBuilderConfigurers = ServiceLoader.load(
+                UndertowBuilderConfigurer.class);
```

Comment:
Minor and I can send up a follow PR after this merged, but we should probably use the `org.jboss.resteasy.spi.PriorityServiceLoader`.

Reply:
DONE